src/pages/intent.tsx

```diff
@@ -0,0 +1,182 @@
+          Select all that apply.
+        </div>
+
+        <div className="govuk-radios">
+          <div className="govuk-radios__item">
+            <input
+              className="govuk-radios__input"
+              id="create_beacon"
+              name="create_beacon"
+              type="radio"
+              value="create_beacon"
+            />
+            <label
+              className="govuk-label govuk-radios__label"
+              htmlFor="create_beacon"
+            >
+              Register a new beacon
+            </label>
+          </div>
+          <div className="govuk-radios__item">
+            <input
+              className="govuk-radios__input"
+              id="change_beacon"
+              name="change_beacon"
+              type="radio"
+              value="change_beacon"
+              disabled
+            />
+            <label
+              className="govuk-label govuk-radios__label"
+              htmlFor="change_beacon"
+            >
+              Update details for existing beacon(s) you have already registered
+            </label>
+          </div>
+          <div className="govuk-radios__item">
+            <input
+              className="govuk-radios__input"
+              id="ownership"
+              name="ownership"
+              type="radio"
+              value="ownership"
+              disabled
+            />
+            <label
+              className="govuk-label govuk-radios__label"
+              htmlFor="ownership"
+            >
+              Change beacon ownership
+            </label>
+          </div>
+          <div className="govuk-radios__item">
+            <input
+              className="govuk-radios__input"
+              id="withdraw"
+              name="withdraw"
+              type="radio"
+              value="withdraw"
+              disabled
+            />
+            <label
+              className="govuk-label govuk-radios__label"
+              htmlFor="withdraw"
+            >
+              Withdraw a beacon
+            </label>
+          </div>
+          <div className="govuk-radios__item">
+            <input
+              className="govuk-radios__input"
+              id="other"
+              name="other"
+              type="radio"
+              value="other"
+              disabled
+            />
+            <label className="govuk-label govuk-radios__label" htmlFor="other">
+              Other
+            </label>
+          </div>
+        </div>
+
+        <br />
+
+        <h2 className="govuk-heading-m">
+          <label htmlFor="event-name">
+            If you selected &apos;Other&apos; please specify your request
+          </label>
+        </h2>
+        <input
+          className="govuk-input govuk-input-disabled"
+          id="other-intent"
+          name="other-intent"
+          type="text"
+        ></input>
+
+        <br />
+        <br />
+
+        <Button buttonText="Continue" />
```

Comment:
Semantically, the button should sit outside the `fieldset` element.  If you take a look at this [question page example on the GDS Design System](https://design-system.service.gov.uk/patterns/question-pages/).

I would also see if we can remove the `<br />` elements as well to see if that resolves the padding.  If it doesn't, I would add styling to the `fieldset` or wrap it in a `div` and add padding that way between the form and the button